etcd-backupper/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.21
+
+RUN apk add etcd-ctl yq
```

Comment:
can we pin the versions? at least the etcd one

Reply:
Yes, working on it.